Source/MaestroInit.cpp

```diff
@@ -462,7 +460,7 @@ void Maestro::InitProj() {
     Make_S_cc(S_cc_old, delta_gamma1_term, delta_gamma1, sold, uold,
               rho_omegadot, rho_Hnuc, rho_Hext, thermal, p0_old, gamma1bar_old,
               delta_gamma1_termbar);
-
+  
```

Comment:
Can you revert the changes to this file (doing by hand is fine)?  The changes were deleting new lines and adding whitespace, neither of which were needed.